proofs/proto/proof.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package proofs;
+
+option go_package = "proofspb";
+option java_multiple_files = true;
+option java_outer_classname = "ProofProto";
+option java_package = "com.proofs";
+
+import "google/protobuf/descriptor.proto";
+
+extend google.protobuf.FieldOptions {
+  bool exclude_from_tree = 2862100;
```

Comment:
Just curious, what is this long number for?

Reply:
Arbitrary, not to clash with any existing options.